lib/index.js

```diff
@@ -41,6 +41,22 @@ bedrock.events.on('bedrock-mongodb.ready', function init(callback) {
 });
 
 /**
+ * Check for the existence of an identity.
+ *
+ * @param id the ID of the identity to check.
+ * @param callback(err, exists) called once the operation completes.
+ */
+api.exists = function(id, callback) {
+  database.collections.identity.find({
```

Comment:
Considered that as well, but since the aim was for optimal performance, and we have not use case for providing an actor I skipped it.  Perhaps use cases that involve an actor could use the `getAll` method.  Not opposed to having a permission check if that is the will of the people.